spec/requests/automation_requests_spec.rb

```diff
@@ -262,7 +262,9 @@
   end
 
   context 'Tasks subcollection' do
-    let(:automation_request) { FactoryGirl.create(:automation_request) }
+    let(:automation_request) { FactoryGirl.create(:automation_request, :requester => @user) }
+    let(:task) { FactoryGirl.create(:miq_request_task, :miq_request_id => automation_request.id) }
```

Comment:
Prefer `:miq_request => automation_request` here - setting the ids explicitly can lead to issues (association callbacks not triggering, etc...)